techeria_controller/templates/ourproducts.html

```diff
@@ -223,109 +223,37 @@
-                                    <div class="product-carousel-price">
-                                        <ins>$899.00</ins> <del>$1099.00</del>
-                                    </div>
-                                </div>
-                                <div class="single-product">
-                                    <div class="product-f-image">
-                                        <img src="/static/watch_css/img/products3.jpg" alt="">
-                                        <div class="product-hover">
-                                            <a href="#" class="add-to-cart-link"><i class="fa fa-shopping-cart"></i> Add to cart</a>
-                                            <a href="single-product.html" class="view-details-link"><i class="fa fa-link"></i> See details</a>
-                                        </div>
-                                    </div>
-
-                                    <h2>Apple iphone 13</h2>
+            <div class="card" style="width: 1000;">
+                <div class="row no-gutters">
 
-                                    <div class="product-carousel-price">
-                                        <ins>$1500.00</ins> <del>$1125.00</del>
-                                    </div>
-                                </div>
-                                <div class="single-product">
-                                    <div class="product-f-image">
-                                        <img src="/static/watch_css/img/samsung2.jpg" alt="">
-                                        <div class="product-hover">
-                                            <a href="#" class="add-to-cart-link"><i class="fa fa-shopping-cart"></i> Add to cart</a>
-                                            <a href="single-product.html" class="view-details-link"><i class="fa fa-link"></i> See details</a>
-                                        </div>
-                                    </div>
-
-                                    <h2><a href="single-product.html">Sony Xperia</a></h2>
-
-                                    <div class="product-carousel-price">
-                                        <ins>$500.00</ins> <del>$225.00</del>
-                                    </div>
-                                </div>
-                                <div class="single-product">
-                                    <div class="product-f-image">
-                                        <img src="/static/watch_css/img/smartphones.jpg" alt="">
-                                        <div class="product-hover">
-                                            <a href="#" class="add-to-cart-link"><i class="fa fa-shopping-cart"></i> Add to cart</a>
-                                            <a href="single-product.html" class="view-details-link"><i class="fa fa-link"></i> See details</a>
-                                        </div>
-                                    </div>
```

Comment:
You could also add the section the price on any other website. And could make a comparison